src/api/.keystone/config.js

```diff
@@ -2351,7 +2351,7 @@ var getApplicationSubmittedEmailTemplateIds = (application) => {
     templateIds.exporter = EXPORTER.SEND_DOCUMENTS.ANTI_BRIBERY;
     templateIds.underwritingTeam = UNDERWRITING_TEAM.NOTIFICATION_ANTI_BRIBERY;
   }
-  const isConnectedWithBuyer = buyer.exporterIsConnectedWithBuyer && buyer.exporterIsConnectedWithBuyer === ANSWERS.YES;
+  const isConnectedWithBuyer = buyer.exporterHasTradedWithBuyer && buyer.exporterHasTradedWithBuyer === ANSWERS.YES;
```

Comment:
does this happen anywhere else? we could have a helper for answerIsYes and answerIsNo

Reply:
Great idea, it happens in only one other place - CSV mapping for underwriters. We have some tickets to fix some CSV things, could create a helper then if that's OK? So we can progress this ticket, has been floating around for a while